src/js/components/header/header.js

```diff
@@ -42,11 +43,15 @@ export class Header extends React.Component {
     this.cookies = new Cookies();
   }
 
-  componentDidUpdate() {
+  componentDidUpdate(prevProps) {
     const sessionId = this.cookies.get('JWT');
-    if ((!sessionId || !this.props.user || !this.props.user.id || !this.props.user.email.length) && !this.state.gettingUser && !this.state.loggingOut) {
+    const { hasTrackingEnabled, isEnterprise, trackingCode, user } = this.props;
+    if ((!sessionId || !user || !user.id || !user.email.length) && !this.state.gettingUser && !this.state.loggingOut) {
       this._updateUsername();
     }
+    if (prevProps.hasTrackingEnabled !== hasTrackingEnabled && trackingCode && !isEnterprise && hasTrackingEnabled) {
```

Comment:
not sure here, why do you need the `!isEnterprise`? It looks weird, because you can have enterprise hosted users.

Reply:
With enterprise setups in general I thought it might get more tricky due to the implications it might have for whatever policies they might have regarding the storage and use of their data... I wouldn't expect this to be as strict for smaller companies. On the other hand - if they have the `_ga` cookie set that could be enough to just continue using it 🤷‍♂️

Reply:
To be honest, I would just skip the distinction and enable GA independently from the plan.
If you want to be sure, we can ask Eystein... anyway, approving the the PR.

Reply:
👌 - then it is now solely based on the presence of the GA code...